components/library/py3c/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Klaus Ziegler
+#
+
+BUILD_STYLE = setup.py
+BUILD_BITS=NO_ARCH
+PYTHON3_ONLY=yes
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           py3c
+COMPONENT_VERSION=        1.3.1
+COMPONENT_SUMMARY=	  port C extensions to Python 3
+COMPONENT_PROJECT_URL=    https://py3c.readthedocs.io/en/latest/
+COMPONENT_SRC=		  $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	  $(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_URL=	  https://github.com/encukou/py3c/archive/refs/tags/v$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:f3e138623a87cde2cd7d8e98b51447e033d43a3f639c5054185c859fa1888727
+COMPONENT_FMRI=           library/py3c
+COMPONENT_CLASSIFICATION= Development/Python
+COMPONENT_LICENSE=        MIT
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/ips.mk
+include $(WS_MAKE_RULES)/setup.py.mk
+
+build: $(BUILD_NO_ARCH)
+
+install: $(INSTALL_NO_ARCH)
+
```

Comment:
build: and install: can be removed as suitable defaults will automatically provided.